pkg/service/schema_test.go

```diff
@@ -10,6 +10,97 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestValidateInputParametersSchema(t *testing.T) {
```

Comment:
There were no tests for `InputParametersSchema`. This adds them. They're mostly the same as the `ObjectPropertySchema` tests, except unconditionally don't allow additional properties.